### DISCUSSION TITLE
Is it possible to use image self-pose information

### DISCUSSION THREAD

**Discussion by Timothy-pop** [NVlabs/instant-ngp#1341] [Q&A] [answered]
Is it possible to use image self-pose information
Example: UAV platform using gimbal and RGB camera scene

my multi-view image has GPS information and gimbal angle information (to get the camera pose), can I not use COLMAP to infer the camera position (because in this case the cameras are widely spaced and the inference will fail) and just use the camera pose information in the image. I would appreciate any reply!

Note: When I import the image into metashape, the camera position and pose are automatically recognized.

**Comment by FlorisE** [comment, marked as answer]
If you can load the data into metashape, how about using metashape 2 instant-ngp? https://github.com/NVlabs/instant-ngp/discussions/438

**Comment by Timothy-pop** [comment]
"Thank you for your reply. I haven't been on Github for a long time and just saw this response. What you mentioned is feasible; it can be achieved through metashape2nerf. However, I've found that the RTK information provided by DJI drones, after image alignment, does not result in very accurate camera positioning. Sometimes, it might even be better not to use the GPS information and directly use the colmap2nerf code that comes with instant-ngp."
